clients/client-python/build.gradle.kts

```diff
@@ -282,13 +282,27 @@ tasks {
       delete("README.md")
       generatePypiProjectHomePage()
       delete("dist")
+      copy {
+        from("${project.rootDir}/licenses") { into("licenses") }
+        from("${project.rootDir}/LICENSE.bin") { into("./") }
+        from("${project.rootDir}/NOTICE.bin") { into("./") }
+        from("${project.rootDir}/DISCLAIMER_WIP.txt") { into("./") }
+        into("${project.rootDir.path}/clients/client-python")
```

Comment:
We can delete `.path` in the `into("${project.rootDir.path}/clients/client-python")`
I test passed use `into("${project.rootDir}/clients/client-python")`.

Reply:
OK, let me update it.